### PR TITLE
doc: remove the Manger documentation from the core ScyllaDB docs

### PR DESCRIPTION
**Pull request by annastuchlik**
In this PR, I have:

- removed the docs for Manager  (including the sources for Manager 2.1 and the upgrade guides).
- added redirects to https://manager.docs.scylladb.com/.
- replaced the internal links with external links to https://manager.docs.scylladb.com/.

### CODE REVIEW

File: docs/_utils/redirects.yaml [hunk 1]
```diff
@@ -1,6 +1,45 @@
 ### a dictionary of redirections
 #old path: new path
 
+### removing the old Monitoring Stack documentation from the ScyllaDB docs
```
Comment (tzach) [inline docs/_utils/redirects.yaml:4]:
```suggestion
### removing the old Scylla Manager documentation from the ScyllaDB docs
```

File: docs/kb/gc-grace-seconds.rst [hunk 2]
```diff
@@ -23,11 +23,11 @@ In addition, you should follow the procedure below in order to avoid data resurr
 
 Resolution
 ----------
-#. Run a :doc:`full repair </operating-scylla/manager/2.1/repair>` for the table in question.
+#. Run a `full repair <https://manager.docs.scylladb.com/stable/repair/index.html>`_ for the table in question.
```
Comment (tzach) [inline docs/kb/gc-grace-seconds.rst:26]:
```suggestion
#. Run a `full repair <https://manager.docs.scylladb.com/stable/repair>`_ for the table in question.
```

Comment (tzach) [reply docs/kb/gc-grace-seconds.rst:26]: Not a must, but shorter URL is better

**Review by tzach — APPROVED** [review]

**Comment by denesb** [comment]
Conflicts, please rebase.

**Comment by annastuchlik** [comment]
@denesb The conflicts are now resolved.

**Comment by denesb** [comment]
> @denesb The conflicts are now resolved.

Queued.

**Comment by annastuchlik** [comment]
@denesb I'm sorry, I had to push three more commits to fix the warnings. Will it prevent the merge?
They can be merged now or I can simply send another commit later on.

**Comment by denesb** [comment]
I'm not exactly sure what will happen. I think these commits will simply not be merged into master when the branch is promoted. Let's wait until promotion runs through. Most likely you will need to open a separate PR with these commits.

**Comment by annastuchlik** [comment]
@scylladb/scylla-maint Please backport to 5.1

**Pull request closed without merging**